include/proxy-wasm/wasm.h

```diff
@@ -51,7 +52,8 @@ class WasmBase : public std::enable_shared_from_this<WasmBase> {
   WasmBase(std::unique_ptr<WasmVm> wasm_vm, std::string_view vm_id,
            std::string_view vm_configuration, std::string_view vm_key,
            std::unordered_map<std::string, std::string> envs,
-           AllowedCapabilitiesMap allowed_capabilities);
+           AllowedCapabilitiesMap allowed_capabilities,
+           FileSystemConfig filesystem);
```

Comment:
same. Otherwise, the entire vector will result in being copied each time.

```suggestion
           std::unique_ptr<FileSystemConfig> filesystem);
```